src/D2L.Bmx/Authenticate.cs

```diff
@@ -38,23 +38,24 @@ internal class Authenticator {
 		if( authState.OktaStateToken is not null ) {
 
 			var mfaOptions = authState.MfaOptions;
-
 			var selectedMfaIndex = PromptMfa( mfaOptions );
 			var selectedMfa = mfaOptions[selectedMfaIndex - 1];
+			string mfaInput = "";
 
+			// TODO: Handle retry
```

Comment:
Let me know if I should handle retry in this task as well

Reply:
Nope, current BMX doesn't support retry so its not necessary for feature parity + better suited for a separate feature PR